src/views/endpoint/autopilot/AutopilotListProfiles.js

```diff
@@ -1,27 +1,58 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { useSelector } from 'react-redux'
 import TenantSelector from '../../../components/cipp/TenantSelector'
 import CippDatatable from '../../../components/cipp/CippDatatable'
 import { CCard, CCardHeader, CCardTitle, CCardBody } from '@coreui/react'
 import { cellBooleanFormatter } from '../../../components/cipp'
-//future version dropdown:
-// const dropdown = (row, rowIndex, formatExtraData) => {
-//   return (
-//     <CDropdown>
-//       <CDropdownToggle size="sm" color="link">
-//         <FontAwesomeIcon icon={faBars} />
-//       </CDropdownToggle>
-//       <CDropdownMenu style={{ position: 'fixed', right: 0, zIndex: 1000 }}>
-//         <CDropdownItem href="#">
-//           <Link className="dropdown-item" to={`/endpoint/autopilot/AutopilotEditProfile}`}>
-//             <FontAwesomeIcon icon={faUser} className="me-2" />
-//             Edit Profile
-//           </Link>
-//         </CDropdownItem>
-//       </CDropdownMenu>
-//     </CDropdown>
-//   )
-// }
+import { CButton } from '@coreui/react'
+import { faCopy, faEye } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { CippOffcanvas } from '../../../components/cipp'
+
+const Offcanvas = (row, rowIndex, formatExtraData) => {
+  const [visible, setVisible] = useState(false)
+
+  const jsonContent = <pre>{JSON.stringify(row, null, 2)}</pre>
+
+  function CopyToClipboard() {
+    const copyText = jsonContent.props.children
+    if (navigator.clipboard) {
+      navigator.clipboard.writeText(copyText).then(
+        () => {
+          console.log('Copied Successfully')
+          console.log({ copyText })
+        },
+        (error) => {
+          console.log(error)
+        },
+      )
+    } else {
+      console.log(document.execCommand('copy'))
+      document.execCommand('copy')
+    }
+  }
+
+  return (
+    <>
+      <CButton size="sm" variant="ghost" color="success" onClick={() => setVisible(true)}>
+        <FontAwesomeIcon icon={faEye} />
+      </CButton>
+      <CippOffcanvas
+        visible={visible}
+        id={row.id}
+        placement="end"
+        className="cipp-offcanvas"
+        hideFunction={() => setVisible(false)}
+        title="JSON (Raw)"
+      >
+        {jsonContent}
+        <CButton size="sm" variant="ghost" color="info" onClick={CopyToClipboard}>
+          <FontAwesomeIcon icon={faCopy} />
+        </CButton>
```

Comment:
I'd put this above the content.

Reply:
I was going to originally, but wanted to mimic the "actions" of CippActionsOffcanvas for that consistent "action" type to be below the content.  Think move it more to the top right?

Reply:
Yeah I think "copy to clipboard" is a top right of the content being copied thing.

Reply:
add `className = "float-end"` to the button.

Reply:
Thanks, was just looking at the documentation ;)